BFB/src/main/java/BFB/GUI/ForcePanel.java

```diff
@@ -39,6 +39,7 @@
 import javax.swing.event.TableModelListener;
 
 public class ForcePanel extends javax.swing.JPanel {
+    private static final long serialVersionUID = 9012868947346629504L;
```

Comment:
Just out of curiosity, how were these generated?

Reply:
VS Code has a 'basic' or 'random' option and just let it pick it at random.